pkgs/by-name/y-/y-cruncher/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  autoPatchelfHook,
+}:
+stdenv.mkDerivation rec {
```

Comment:
consider switching to the finalAttrs pattern. It's the norm now and most reviewers require it, but I don't explicitly require it yet.